gradlew.bat

```diff
@@ -1,84 +1,84 @@
-@if "%DEBUG%" == "" @echo off
```

Comment:
Were the line endings in this file broken? If not, could you please fix the line endings?

Reply:
Actually, this is default generated file  of `./gradlew wrapper --gradle-version=4.7`
Should I change it anyway?

Reply:
I think Git converted it from win to nix endings, but the code looks the same

Reply:
Ok, I looked the old `graldew.bat` and the new one up in the hex-editor:

This is really just one line without line-break
```@if "%DEBUG%" == "" @echo off```

The "old" file had just `LF` ( `0A` ) as line ending (NL line feed)
The "new" file, has `CR` + `LF` ( `0D` + `0A`) as line ending (carriage return + NL line feed)

I guess, the new file is more accurate for windows than the old is.